enterprise/api/v1/views.py

```diff
@@ -77,7 +78,7 @@ class EnterpriseReadWriteModelViewSet(EnterpriseModelViewSet, viewsets.ModelView
     Base class for all read/write Enterprise model view sets.
     """
 
-    permission_classes = (permissions.IsAuthenticated, IsServiceUserOrReadOnly,)
+    permission_classes = (permissions.IsAuthenticated, permissions.DjangoModelPermissions, IsServiceUserOrReadOnly,)
```

Comment:
Yeah, it's basically because these are *model* permissions and not every view set inheriting from `EnterpriseViewSet` is necessarily going to be a model's view set (we have `EnterpriseModelViewSet` for that).

Reply:
Ah, OK. Thanks for the details!

Reply:
Didn't we decide we no longer need `IsServiceUserOrReadOnly` now that we are using `DjangoModelPermissions`?

Reply:
@douglashall Yep sorry, you're right about that.